docling/datamodel/base_models.py

```diff
@@ -68,6 +70,7 @@ class OutputFormat(str, Enum):
         "application/vnd.openxmlformats-officedocument.presentationml.presentation",
     ],
     InputFormat.HTML: ["text/html", "application/xhtml+xml"],
+    InputFormat.PUBMED: ["application/nxml"],
```

Comment:
I don't think this is valid mime type. if it is not supported as official mimetype we might have to add it to the extension list in docling-core